template/Makefile.j2

```diff
@@ -73,3 +73,6 @@ regenerate-charts: chart-clean compile-chart
 build: regenerate-charts helm-package docker-build
 
 publish: build docker-publish helm-publish
+
+run-dev:
+	nix run -f. tilt -- up --port 543{[operator_index}]
```

Comment:
This is going to go from `5439` to `54310`, which is pretty surprising. Maybe `--port {[5430 + operator_index}]` would be a better fit?

Reply:
It would indeed, didn't know that was possible, which is why I was willing to live with that jump :)